integration/tests/cook/test_multi_user.py

```diff
@@ -389,18 +389,20 @@ def trigger_preemption(self, pool):
         5. Submit a job, J2, from X with 0.1 cpu and priority 100
         6. Wait until J1 is preempted (to make room for J2)
         """
-        admin = self.user_factory.admin()
         user = self.user_factory.new_user()
         all_job_uuids = []
         try:
             large_cpus = util.get_default_cpus()
             small_cpus = large_cpus / 10
-            with admin:
-                # Lower the user's cpu share and quota
-                util.set_limit(self.cook_url, 'share', user.name, cpus=small_cpus, pool=pool)
-                util.set_limit(self.cook_url, 'quota', user.name, cpus=large_cpus, pool=pool)
+            with self.user_factory.admin():
+                # Reset the user's share and quota
+                util.reset_limit(self.cook_url, 'share', user.name, reason=self.current_name(), pool=pool)
+                util.reset_limit(self.cook_url, 'quota', user.name, reason=self.current_name(), pool=pool)
```

Comment:
Can we assert on these responses as well?

Reply:
Good call, done

Reply:
Turns out that if you try to delete share for a user that doesn't have it defined, you get a 500. Will need to revisit this tomorrow.

Reply:
Sigh

Reply:
I changed this to set the limit to the `default` limit instead of deleting it.

Reply:
I also ended up marking the preemption tests as `xfail`, see the PR description